hack/test-cmd.sh

```diff
@@ -785,12 +804,18 @@ __EOF__
     kube::test::get_object_assert 'services mock' "{{${labels_field}.status}}" 'replaced'
     kube::test::get_object_assert 'rc mock' "{{${labels_field}.status}}" 'replaced'
     # Command
-    kubectl label -f $file labeled=true "${kube_flags[@]}"
+    # We need to set --overwrite, because otherwise, if the first attempt to run "kubectl label" 
```

Comment:
I discussed with @JanetKuo and she said the purpose of this test is to test if kubectl label/annotate works with multiple resource, and we don't care about if --overwrite is set.

Do you think my comment makes sense? I need to reword it if you can't interpret it.